src/pharmpy/tools/iivsearch/tool.py

```diff
@@ -423,12 +423,12 @@ def flatten_list(lst):
 def validate_input(
     algorithm, iiv_strategy, rank_type, model, keep, strictness, correlation_algorithm
 ):
-    if keep:
+    if keep and model:
         for parameter in keep:
             try:
                 has_random_effect(model, parameter, "iiv")
             except KeyError:
-                raise ValueError(f"Parameter {parameter} has no iiv.")
+                warnings.warn(f"Parameter {keep} has no iiv and is ignored")
```

Comment:
Why is this a warning now?

Reply:
As "CL" might not always be part of the model (but is now the default value) this needed to be changed. Otherwise we might raise with the default values